### PR TITLE
Feat/add pizza hut be

### PR DESCRIPTION
**Pull request by elsa-alves-monteiro**

### CODE REVIEW

File: locations/spiders/pizza_hut_be.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    def parse(self, response):
+        for data in response.json():
+            yield scrapy.Request(f"https://api.pizzahut.be/stores/{data.get('id')}", callback=self.parse_store)
+
+    def parse_store(self, response):
+        oh = OpeningHours()
+        store = response.json().get("store")
+        for ohs in store.get("openingHours"):
+            open, close = ohs.get("shift1").split("/")
```
Comment (Cj-Malone) [inline locations/spiders/pizza_hut_be.py:23]: Does "shift**1**" imply there is a "shift**2**"? Is it worth collecting?

File: locations/spiders/pizza_hut_be.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            "ref": store.get("code"),
+            "addr_full": " ".join(
+                [
+                    address_details.get("address1"),
+                    address_details.get("address2"),
+                    address_details.get("city"),
+                    address_details.get("zipCode"),
+                ]
+            ),
+            "street_address": address_details.get("address2"),
```
Comment (Cj-Malone) [inline locations/spiders/pizza_hut_be.py:37]:
```suggestion
            "street_address": ", ".join(filter(None, [address_details.get("address1"), address_details.get("address2")])),
```

File: locations/spiders/pizza_hut_be.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        oh = OpeningHours()
+        store = response.json().get("store")
+        for ohs in store.get("openingHours"):
+            open, close = ohs.get("shift1").split("/")
+            oh.add_range(day=DAYS[ohs.get("dayNum") - 1], open_time=open, close_time=close)
+
+        address_details = store.get("address")
+        properties = {
+            "ref": store.get("code"),
+            "addr_full": " ".join(
```
Comment (Cj-Malone) [inline locations/spiders/pizza_hut_be.py:29]: IMO a comma separator is better. For people and for computers.

```suggestion
            "addr_full": ", ".join(
```

File: locations/spiders/pizza_hut_be.py (hunk 1 above)
Comment (Cj-Malone) [reply locations/spiders/pizza_hut_be.py:23]: OK bad data. Yeah ignore it, thanks for checking.

**Comment by Cj-Malone** [comment]
@elsa-alves-monteiro Would you mind enabling ["Allow edits from maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork). I have a few minor things I'd like to change.

**Comment by elsa-alves-monteiro** [comment]
> @elsa-alves-monteiro Would you mind enabling ["Allow edits from maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork). I have a few minor things I'd like to change.

I don't have this option since the repo was forked inside an organization. Do you have another way or do you prefer me to fork the project again on my user and create a new pull request from there ?

**Comment by Cj-Malone** [comment]
Going forward it'd be nice, but it's fine.

I've added categories, fixed the phone numbers, filter out `address2` when it's null, and used 1 API instead of 2.

```python
import scrapy

from locations.categories import Categories, Extras, apply_category, apply_yes_no
from locations.hours import DAYS, OpeningHours
from locations.items import Feature


class PizzaHutBESpider(scrapy.Spider):
    name = "pizza_hut_be"
    item_attributes = {"brand": "Pizza Hut", "brand_wikidata": "Q191615"}
    start_urls = ["https://api.pizzahut.be/stores"]
    custom_settings = {"ROBOTSTXT_OBEY": False}

    def parse(self, response):
        for data in response.json()["stores"]:
            yield scrapy.Request(f"https://api.pizzahut.be/stores/{data.get('code')}", callback=self.parse_store)

    def parse_store(self, response):
        oh = OpeningHours()
        store = response.json().get("store")
        for ohs in store.get("openingHours"):
            open, close = ohs.get("shift1").split("/")
            oh.add_range(day=DAYS[ohs.get("dayNum") - 1], open_time=open, close_time=close)

        address_details = store.get("address")
        properties = {
            "ref": store.get("code"),
            "addr_full": ", ".join(
                filter(
                    None,
                    [
                        address_details.get("address1"),
                        address_details.get("address2"),
                        address_details.get("city"),
                        address_details.get("zipCode"),
                    ],
                )
            ),
            "street_address": ", ".join(
                filter(None, [address_details.get("address1"), address_details.get("address2")])
            ),
            "name": store["name"],
            "postcode": address_details.get("zipCode"),
            "city": address_details.get("city"),
            "website": f"https://restaurants.pizzahut.be/fr/restaurant/{store.get('slug')}",
            "lat": address_details.get("coordinates").get("latitude"),
            "lon": address_details.get("coordinates").get("longitude"),
            "phone": store.get("contact").get("phone").replace("/", ""),
            "opening_hours": oh,
        }

        if store["storeType"] == "FSR":
            apply_category(Categories.RESTAURANT, properties)
        elif store["storeType"] == "DELCO":
            apply_category(Categories.FAST_FOOD, properties)

        apply_yes_no(Extras.TAKEAWAY, properties, store["takeoutStatus"]["isActive"])
        apply_yes_no(Extras.DELIVERY, properties, store["deliveryStatus"]["isActive"])

        yield Feature(**properties)
```

**Comment by elsa-alves-monteiro** [comment]
@Cj-Malone, I commited your changes

**Comment by Cj-Malone** [comment]
Thank you

**Pull request merged**